egs/wsj/s5/steps/nnet3/nnet3_train_lib.py

```diff
@@ -175,25 +183,28 @@ def SplitData(data, num_jobs):
                                                                   num_jobs = num_jobs))
 
 def ParseModelConfigVarsFile(var_file):
+    variables = {}
```

Comment:
guys- I don't see why you are adding this huge mechanism.  Didn't the old method work fine?  And didn't we only want to parse a few specific variables and ignore the rest?
Is there a specific projected use for this?